packages/runtime/src/templates/handlerUtils.ts

```diff
@@ -188,3 +190,19 @@ export const getNextServer = (): NextServerType => {
   }
   return NextServer
 }
+/**
+ * Prefetch requests are used to check for middleware redirects, and shouldn't trigger SSR.
+ */
+export const getPrefetchResponse = (event: HandlerEvent, mode: string): HandlerResponse | false => {
+  if (event.headers['x-middleware-prefetch'] && mode === 'ssr') {
+    return {
+      statusCode: 200,
+      body: '{}',
```

Comment:
Yes, this is the correct value. https://github.com/vercel/next.js/blob/canary/packages/next/server/base-server.ts#L1060-L1068